docs/conf.py

```diff
@@ -30,6 +30,15 @@
           file=sys.stderr)
 
 
+from mock import Mock as MagicMock
```

Comment:
Actually I just followed http://docs.readthedocs.org/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules so I am not sure why they are not just doing that.